README.md

```diff
@@ -1,22 +1,22 @@
-# eirini-release
+# Eirini Release
 
-This is a `helm` release for [eirini](https://code.cloudfoundry.org/eirini).
+This is a `helm` release for Project [Eirini](https://code.cloudfoundry.org/eirini).
 
-- This is an **experimental** release and is still considered _work in progress_.
+**NOTE**: This is an **_experimental_** release and is still considered _work in progress_.
 
-## Deploy Eirini
+## Deploy Eirini with Containerized Cloud Foundry (SCF)
 
 If you want to deploy a full containerized CF (`SCF`) with `Eirini` you should follow the [SCF + Eirini documentation](./scf/README.md)
 
+## Deploy Standalone Eirini
+
 If you want to deploy _only_ Eirini on a Kube cluster, you can follow the instruction below:
 
-### Prereqs
+### Prerequisites
 
 Beside a Kubernetes cluster you should have:
 
-- [helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
-
-#### Deploy
+### Deploy
```

Comment:
Isn't helm still a prereq?

Reply:
put it back. I think it dropped out as I was trying to figure out the compatible helm version